ProcessMaker/modeler
# Fixing Loading Issue and Enhancing Parsing for PM Blocks in Process Reloading and Importing

## Issue & Reproduction Steps

This PR effectively resolves a loading issue that occurs during reloading a process when a PM Block is configured. Similarly, it tackles an issue arising when importing a process containing a PM Block.

Previously, when a PM Block was loaded, its node type would erroneously convert to a subprocess, disrupting the configuration of the custom inspector panel for that particular PM Block.

Upon investigation, the problem was traced back to the Modeler's createNode method. The underlying parser in this method was inadvertently defaulting to the `processmaker-modeler-call-activity` instead of the required custom pm-block node.


## Test Case A:
1. Create a PM Block
2. Create a Process
3. Add the PM Block to the process
4. Save the process
5. Reload the process (you may have to reload multiple times)

## Test Case B:
1. Create a PM Block
2. Create a Process
3. Add the PM Block to the process
4. Save the process
5. Export the process
6. Reimport the process

Expected behavior: 
The PM Block loads the correct node type

Actual behavior: 

The PM Block is converted to the subprocess node type

## Solution
- Implement the correct parsing mechanism for PM Blocks. 

## How to Test
Test the steps above

ci:package-pm-blocks:observation/FOUR-8887
.

## Related Tickets & Packages
- PM Blocks PR
- Ticket: [FOUR-8887](https://processmaker.atlassian.net/browse/FOUR-8887) & [FOUR-9321](https://processmaker.atlassian.net/browse/FOUR-9321)

## Code Review Checklist


[FOUR-8887]: https://processmaker.atlassian.net/browse/FOUR-8887?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
